projects/1.20/assets/refurbished-furniture/refurbished_furniture/lang/zh_cn-base.json

```
@@ -0,0 +1,211 @@
+{
+    "itemGroup.refurbished_furniture": "MrCrayfish的家具模组",
+    "filterCategory.refurbished_furniture.general": "常规",
+    "filterCategory.refurbished_furniture.general.desc": "所有基础家具，椅子、桌子等等",
+    "filterCategory.refurbished_furniture.bedroom": "卧室",
+    "filterCategory.refurbished_furniture.bedroom.desc": "床、书桌、梳妆台等等",
+    "filterCategory.refurbished_furniture.kitchen": "厨房",
+    "filterCategory.refurbished_furniture.kitchen.desc": "橱柜、台面、电器等等",
+    "filterCategory.refurbished_furniture.bathroom": "浴室",
+    "filterCategory.refurbished_furniture.bathroom.desc": "马桶、洗手盆等等",
+    "filterCategory.refurbished_furniture.electronics": "电子产品",
+    "filterCategory.refurbished_furniture.electronics.desc": "灯具、电脑、发电机等等",
+    "filterCategory.refurbished_furniture.outdoors": "户外",
+    "filterCategory.refurbished_furniture.outdoors.desc": "邮箱、树篱、围栏等等",
+    "filterCategory.refurbished_furniture.storage": "存储",
+    "filterCategory.refurbished_furniture.storage.desc": "所有带有存储功能的家具和装饰",
+    "filterCategory.refurbished_furniture.food": "食物",
+    "filterCategory.refurbished_furniture.food.desc": "食物和烹饪原料",
+    "filterCategory.refurbished_furniture.items": "物品",
+    "filterCategory.refurbished_furniture.items.desc": "所有物品，锅铲、平底锅、吐司等等",
+    "death.attack.refurbished_furniture.ceiling_fan": "%1$s被吊扇切成了碎片",
+    "death.attack.refurbished_furniture.ceiling_fan.player": "%1$s被吊扇切成了碎片",
+    "block.refurbished_furniture.workbench": "工作台",
+    "block.refurbished_furniture.frying_pan": "平底锅",
+    "block.refurbished_furniture.post_box": "邮筒",
+    "block.refurbished_furniture.doorbell": "门铃",
+    "block.refurbished_furniture.recycle_bin": "回收桶",
+    "block.refurbished_furniture.plate": "盘子",
+    "block.refurbished_furniture.stone_stepping_stones": "石头踏脚石",
+    "block.refurbished_furniture.granite_stepping_stones": "花岗岩踏脚石",
+    "block.refurbished_furniture.diorite_stepping_stones": "闪长岩踏脚石",
+    "block.refurbished_furniture.andesite_stepping_stones": "安山岩踏脚石",
+    "block.refurbished_furniture.deepslate_stepping_stones": "深板岩踏脚石",
+    "block.refurbished_furniture.television": "电视机",
+    "block.refurbished_furniture.computer": "电脑",
+    "block.refurbished_furniture.door_mat": "门垫",
+    "block.refurbished_furniture.milk": "奶",
+    "item.refurbished_furniture.spatula": "锅铲",
+    "item.refurbished_furniture.knife": "刀",
+    "item.refurbished_furniture.package": "包裹",
+    "item.refurbished_furniture.wrench": "扳手",
+    "item.refurbished_furniture.television_remote": "电视遥控器",
+    "item.refurbished_furniture.bread_slice": "面包片",
+    "item.refurbished_furniture.toast": "吐司",
+    "item.refurbished_furniture.sweet_berry_jam": "甜浆果果酱",
+    "item.refurbished_furniture.sweet_berry_jam_toast": "甜浆果果酱吐司",
+    "item.refurbished_furniture.glow_berry_jam": "发光浆果酱",
+    "item.refurbished_furniture.glow_berry_jam_toast": "发光浆果酱吐司",

```

```suggestion
    "item.refurbished_furniture.glow_berry_jam": "发光浆果果酱",
    "item.refurbished_furniture.glow_berry_jam_toast": "发光浆果果酱吐司",
```
与甜浆果果酱统一命名格式
是我之前没说清楚